weed/storage/volume.go

```diff
@@ -25,8 +25,7 @@ type Volume struct {
 	Collection         string
 	DataBackend        backend.BackendStorageFile
 	nm                 NeedleMapper
-	tmpNm              *NeedleMap
-	tmpLnm             *LevelDbNeedleMap
+	tmpNm              NeedleMapper
```

Comment:
You can add an interface, e.g., TempNeedleMapper,
```
type TempNeedleMapper interface {
    NeedleMapper
    some_additional_functions()
}
```

Reply:
Sorry maybe I don't really get your point.
Do I just replace NeedleMapper with TempNeedleMapper. If so, no more modification will be need,  the current logic is OK.
Or  may I need to implement "some_additional_functions()" for incremental loading?

Reply:
Use it to avoid reflection

Reply:
Done